File: week-1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

// Prints given amount of symbols
void print_symbols(char symbol, int amount)
{
    for (int i = 0; i < amount; i++)
    {
        printf("%c", symbol);
    }
}

int main(void)
{
    int height;

    // Ask for pyramid height. It should be between 1 and 8, otherwise asks untill get right integer.
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    // Loop for printing pyramid
    for (int line = 1; line <= height; line++)
    {
        int spaces = height - line;

        print_symbols(' ', spaces);
        print_symbols('#', line);
        print_symbols(' ', 2);
        print_symbols('#', line);
        printf("\n");
    }
}

File: week-6/sentimental-mario-more/mario.py
from cs50 import get_int

while True:
    height = get_int("Type height of the piramid: ")
    if 0 < height <= 8:
        break
    print("Height should be more than 0 and less than 9")

for i in range(1, height + 1):
    spaces = height - i
    print(" " * spaces, end="")
    print("#" * i, end="  ")
    print("#" * i)